src/components/PickupBox/PickupBox.jsx

```diff
@@ -0,0 +1,172 @@
+import { React, useState } from 'react';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionPanel,
+  AccordionButton,
+  AccordionIcon,
+  ChakraProvider,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+} from '@chakra-ui/react';
+
+import PropTypes from 'prop-types';
+import styles from './PickupBox.module.css';
+import RejectBoxPopup from '../AlertPopups/RejectBoxPopup/RejectBoxPopup';
+import ApproveBoxIcon from '../BoxIcons/ApproveBoxIcon.svg';
+import RejectBoxIcon from '../BoxIcons/RejectBoxIcon.svg';
+import PickupBoxIcon from '../BoxIcons/PickupBoxIcon.svg';
+import ImageVector from '../BoxIcons/ImageVector.svg';
+import { FYABackend } from '../../common/utils';
+
+const PickupBox = ({
+  approved,
+  boxID,
+  boxHolderName,
+  boxHolderEmail,
+  zipCode,
+  picture,
+  date,
+  status,
+  rejectionReason,
+  fetchBoxes,
+  pickup,
+}) => {
+  // A state for determining whether or not the rejectBoxPopup is open
+  // This state is set true when the reject button is clicked
+  const [rejectBoxPopupIsOpen, setRejectBoxPopupIsOpen] = useState(false);
+
+  // A function that updates the approved boolean in the backend and refreshes all boxes that are under review
+  // This method is called when the approve box icon is clicked
+  const approvePickupBoxFromUR = async id => {
```

Comment:
nit. Please type out UnderReview in the function name. Took me a second to figure out what UR was referring to lol